config/test/test.config.js

```diff
@@ -12,15 +12,27 @@ module.exports = {
     },
     restoreMocks: true,
     resetMocks: true,
-    testEnvironment: "jest-environment-node",
-    testMatch: ["<rootDir>/**/*.test.js"],
+    testEnvironment: "jsdom",
+    testMatch: [
+        "<rootDir>/**/*.test.js",
+        "<rootDir>/**/*_test.js",
+        "<rootDir>/**/*_test.jsx",
+    ],
+    timers: "fake",
     setupFilesAfterEnv: [
         "jest-extended/all",
         "<rootDir>/config/test/test-setup.js",
+        "<rootDir>/config/test/custom-matchers.js",
     ],
     moduleNameMapper: {
         "^@khanacademy/perseus(.*)$":
             "<rootDir>/packages/perseus$1/src/index.js",
+        // TODO: generate this mapping automatically
+        hubble: "<rootDir>/vendor/hubble/hubble.js",
+        jsdiff: "<rootDir>/vendor/jsdiff/jsdiff.js",
+        raphael: "<rootDir>/vendor/raphael/raphael.js",
```

Comment:
I've wanted to do a map => object so many times.  I think it'd even be useful within perseus as we start to replace underscore. 

Something like this would be useful here:
```js
...objMap(["nibble", "jsdiff", "raphael"], key => {
    return [
        key, 
        `<rootDir>/vendor/${key}/${key}.js`,
    ];
}

```

No action required.